readme.md

```diff
@@ -94,7 +105,7 @@ How is the concept of OAuth related to a valet key?
 
 Your Answer:
 ```text
-
+OAuth is an agreed-upon set of standards for logging in through a third party service. Like a valet service OAuth uses special keys to identify that people are who they say they are.  
 ```
```

Comment:
Valet keys allow you to give someone the keys to your car without full access (e.g. they can't open the trunk or glovebox). Similarly, OAuth allows you to share your account on a 3rd party provider (e.g. Facebook, Twitter, Google), without given the consumer app your password, or necessarily full access to your account, such as reading / modifying  your data, posting on your behalf, etc.